Automattic/wp-calypso
# Update images

This PR removes some odd visual artifacts found in the Checklist banner illustrations.

## Before
![screen shot 2017-12-12 at 10 03 47 pm](https://user-images.githubusercontent.com/6981253/33919796-a2c41c9e-df88-11e7-8362-b0250b94cf55.png)

![screen shot 2017-12-12 at 10 07 16 pm](https://user-images.githubusercontent.com/6981253/33919857-e3adc296-df88-11e7-9db8-ec64f90f19cf.png)

![screen shot 2017-12-12 at 10 08 31 pm](https://user-images.githubusercontent.com/6981253/33919919-31b144d6-df89-11e7-910b-73e883020140.png)

![screen shot 2017-12-12 at 10 09 37 pm](https://user-images.githubusercontent.com/6981253/33919922-31e113aa-df89-11e7-9088-cbb86c1c7983.png)

## After
![screen shot 2017-12-12 at 10 03 26 pm](https://user-images.githubusercontent.com/6981253/33919816-b40d2c8e-df88-11e7-9644-2ff514fb85cc.png)

![screen shot 2017-12-12 at 10 01 08 pm](https://user-images.githubusercontent.com/6981253/33919834-c28e880c-df88-11e7-9212-c448837d47db.png)

![screen shot 2017-12-12 at 10 08 52 pm](https://user-images.githubusercontent.com/6981253/33919920-31c1bbb8-df89-11e7-9401-ff7df02421c5.png)

![screen shot 2017-12-12 at 10 09 20 pm](https://user-images.githubusercontent.com/6981253/33919921-31d05628-df89-11e7-8f4e-7ce41d2269db.png)

## Testing
* Create a new site.
* Visit /stats for your new site and mark tasks complete from the checklist to see different banners.

cc @taggon
